lib/hades_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule HadesWeb.UserController do
```

Comment:
Let's remove this controller for now, what do you think?
Then we can do the web part of users in another PR as we talked